cmake/modules/FindAvahi.cmake

```diff
@@ -3,33 +3,29 @@
 # ---------
 # Finds the avahi library
 #
-# This will define the following target:
+# This will define the following targets:
 #
-#   Avahi::Avahi - The avahi client library
-#   Avahi::AvahiCommon - The avahi common library
+#   ${APP_NAME_LC}::Avahi - The avahi client library
```

Comment:
looks like ${APP_NAME_LC} is set to `kodi` for some reason, while it should be `Avahi` ?

```
-- Internal TexturePacker will be executed during build
-- Found Alsa: /usr/lib/x86_64-linux-gnu/libasound.so (found suitable version "1.2.8", minimum required is "1.0.27") 
************** APP_NAME_LC is set to *************** : kodi
-- Found Avahi: /usr/lib/x86_64-linux-gnu/libavahi-client.so (found version "0.8") 
-- Could NOT find CAP (missing: CAP_LIBRARY CAP_INCLUDE_DIR)

```

thus resulting in the error @mackal is reporting

Reply:
https://github.com/xbmc/xbmc/pull/25197/files#diff-38a3c2ed75aedaad964f1d8f0efc0531fa78d52cb7a7a177d7f64da9390e5403R49 I think the issue is that line is referencing something that is now called kodi::AvahiCommon